pkg/virt-controller/services/template.go

```diff
@@ -985,8 +995,7 @@ func (t *templateService) RenderHotplugAttachmentTriggerPodTemplate(volume *v1.V
 					},
 					SecurityContext: &k8sv1.SecurityContext{
 						SELinuxOptions: &k8sv1.SELinuxOptions{
-							Level: "s0",
```

Comment:
Do you want to leave this one out of this PR similarly to `RenderHotplugAttachmentPodTemplate`?